bokeh/document.py

```diff
@@ -346,6 +347,10 @@ def to_json_string(self):
             'roots' : {
                 'root_ids' : root_ids,
                 'references' : self._references_json(root_references)
+            },
+            'version' : {
+                'full' : __version__,
+                'base' : __base_version__
```

Comment:
the use I imagine here is a future compatability check or hack. Using 'base' is probably right for that kind of thing. But we could as easily do the parsing from full to base later, or any other parsing later, so maybe this should simply be `'version':__version__` and leave it at that?